apps/web/src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue

```diff
@@ -37,7 +37,7 @@
                               :class="{inherit: inheritableProperties.includes(propertyName)}"
                         >{{ $t('DASHBOARDS.CUSTOMIZE.ADD_WIDGET.INHERIT') }}</span>
                         <p-toggle-button :value="inheritableProperties.includes(propertyName)"
-                                         :disabled="widgetOptionsJsonSchema.properties?.[propertyName]?.disabled"
+                                         :disabled="widgetOptionsJsonSchema.properties?.[propertyName]?.disabled || fixedProperties.includes(propertyName)"
```

Comment:
description: disable toggle button when it's fixed options(ex. group by)

![스크린샷 2023-06-01 오후 1 47 10](https://github.com/cloudforet-io/console/assets/18563857/3de61406-9592-40d9-863c-f3bc8c938e5e)